NosTests/NoteParserTests.swift

```diff
@@ -2,8 +2,90 @@ import CoreData
 import XCTest
 import Dependencies
 
+// swiftlint:disable file_length
```

Comment:
One thing we might do to try to keep this file length below our threshold is to move the `AttributedString` extension to another file. Not sure if we have anything else like that we could move out, but I'd love it if we could keep files smallish.

Maybe even a new file that contains an `extension NoteParserTests` would help, and it could contain a set of tests that are all related to each other somehow (events, like `testContentWithUntaggedEvent` and similar?).

Reply:
See my comment below.